src/stable/lib/state.ts

```diff
@@ -185,21 +185,35 @@ globalThis._azleDispatch = (action: Action): void => {
         globalThis.process !== undefined &&
         globalThis.process.env.AZLE_RECORD_ACTIONS === 'true'
     ) {
-        globalThis._azleActions?.push(action);
+        if (globalThis._azleActions !== undefined) {
+            globalThis._azleActions.push(action);
+        }
     }
 
     if (action.type === 'DELETE_AZLE_REJECT_CALLBACK') {
-        delete globalThis._azleRejectCallbacks[action.payload];
+        if (globalThis._azleRejectCallbacks !== undefined) {
```

Comment:
But use more of a guard if statement. In this case for example, check first for the undefined case and throw inside of that block. Then run the deletion and return statement outside of the if statement, with no else statement.

Reply:
Repeat this everywhere you changed code in this file